### ISSUE TITLE
Allow multiple editable files to be open in the NovelWriter UI simultaneously

### ISSUE THREAD

**Issue by GrifVindh** [vkbo/novelWriter#1432]
When I'm writing I usually have at least two different documents open:

 - the story I'm actively writing
 - a file of notes that list important story points that haven't been revealed yet, character names, locations, etc

Sometimes if the story is complex I'll have more than one of these note files open, and I'm often updating them as I write.

The current problem with novelwriter -- and to be clear, it's not an issue with the application as much as it's just not compatible with my workflow -- is that it allows you to only have one editable pane. Ideally if I were going to create a project, at it's simplest I'd have two files: Draft and Notes. And I'd have Draft open in one pane where I write, and Notes open where I add character names and brief descriptions as I add them to the story, and settings and location names (name of a restaurant, bar, place of business, organization, etc.) so that I can keep track of them during the story.

Currently in novelwriter I can have both files but I can only edit one at a time. I can *display* the other but it's read only. This means in order to work the way I do I'd essentially have to have the draft open in novelwriter and the notes file open in a separate text editor, which seems to defeat the purpose of an app like novelwriter to begin with.

Allowing multiple editable files to be open within novelwriter at the same time and being able to hop between panes to update each as needed would be an enormous help.

**Comment by vkbo** [comment]
Yes, this is a limitation that bothers me from time to time as well. Since the editor/viewer layout is central to how noverlWriter is built, it is pretty complicated to redesign. I plan to make some improvements that will at least help with this, even if it won't exactly solve the issue. One is to add the option to view multiple notes at the same time in the viewer (see #102), and another is to add a better way to quickly jump between files in the editor.

The latter is discussed in #1292, and I have some ideas to also make it possible to open references in the editor as well as perhaps display a short list of recently edited documents that would make it easier to switch between them when you're working on a set of related ones, like a scene and the character files of the same scene.

After having worked on novelWriter for nearly 5 years now, I have a bunch of ideas I cannot really implement due to choices made along the way. The issue you raise here is one of them, Therefore I have a side project with a new app where I experiment with different approaches, and use C++ instead of Python, which lets me customise the Qt framework a lot more without taking a massive performance hit. Unfortunately, my time is limited so progress is very slow.

**Comment by vkbo** [comment]
I'm closing this as it is effectively replaced by #2855 where I plant to make both panels editable.
